### DISCUSSION TITLE
Video Not playing

### DISCUSSION THREAD

**Discussion by rakesh4637** [community/community#18811] [Other Feature Feedback, Questions, & Ideas]
Video not Playing, only audio track playing please fix it thanks.

**Comment by vutrananpho** [comment]
Can you be more specific about the video that is not playing?

**Comment by rakesh4637** [comment]
None of the video are playing

**Comment by queenofcorgis** [comment]
Hi there - is this happening in a specific repository? If you could link some examples that would be helpful!
